requirements.txt

```diff
@@ -12,3 +12,7 @@ pif>=0.8.2,<0.9
 xxhash>=2.0.0,<2.1
```

Comment:
Please pin all direct dependencies (and markupsafe if required)

Reply:
Tracked in #179